src/api/ci/jenkinsClient.ts

```
@@ -95,6 +100,31 @@ export class JenkinsClient {
     });
   }
 
+  /**
+   * Get information about a job
+   * @param jobPath The path to the job (can include folders, e.g., "folder/job")
+   */
+  public async getJob(jobPath: string): Promise<any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)